zlp-scheduler/app/controllers/student_controller.rb

```diff
@@ -82,12 +82,14 @@ def create_schedule
       @schedule = Schedule.new
       @schedule.update_attributes(:name => params[:schedule][:name])
       @user.schedules.push(@schedule)
+      warning_word = ""
       7.times do |n|
         subj_symb = "dept_id_#{n+1}".to_sym
         number_symb = "course_num_id_#{n+1}".to_sym
         section_symb = "section_num_id_#{n+1}".to_sym
         check_symb = "mand_#{n+1}".to_sym
-        if !(params[section_symb] == "")
+        #if !(params[section_symb] == "")
```

Comment:
some commented out code should be removed
(as well as commented out code in other files)